zer0-os/zOS
# feat: match container to Figma

### What does this do?

In general, it matches the zOS container to Figma. More specifically:

- Changes colours to match Figma
- Adds "3 section" positioning to each of the side bars
- Stretched header over the whole app sandbox
- Switched `Connect` button to zUI `Button`
- Added network to sidebar (hardcoded to Wilder World currently)
- Generally just made positioning of UI a bit tighter

_Note_
This doesn't match Figma perfectly, as it's not the primary focus right now. We just wanted to get the colours right so it's obvious everything belongs to one design system - this communicates the idea behind and the values of zDS/zUI. This will require another iteration later down the line when we have more bandwidth, but for now I think it looks tidy enough.

### Why are we making this change?

- zOS container didn't match Figma
- Needed zOS container to match for consistency w zApps being delivered this week

### How do I test this?

- Open any page
- Check the design against Figma


### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
 -

  1. How will this affect performance?
-

  1. Does this change any APIs?
No
